CHANGES.md

```diff
@@ -16,6 +16,7 @@ Grammars:
   - enh(clojure) Add `regex` mode to regex literal
   - fix(clojure) Remove inconsistent/broken highlighting for metadata
   - enh(clojure) Add `punctuation` mode for commas.
+- enh(nsis) Update defines pattern to allow `!` (#3417) [idleberg][]
```

Comment:
Please add your name around line ~30 also.  Each version needs the name repeated, even if you've contributed in the past.